cdn/dev/js/kmwlive.js

```diff
@@ -241,6 +240,28 @@ $(document).ready(function() {
   window.setInterval(refreshButtons,200);  
 });
 
+// The KeymanWeb script will have loaded by this point, though initialization may be another matter.
+// Promises only started being supported in KMW 13 (keymanapp/keyman#1432), but this site supports
+// earlier versions, too, so we can't rely on their presence.
+$(window).on("load", function() { 
+  // Focus on message box
+  setTimeout(function(){
+    if (!!$('.messageBox').length) {
+      getKeymanWeb().moveToElement('message');
+    }
+    // On touch devices, this is necessary (but not sufficient) for ClipboardJS compatibility.
+    // Must take affect AFTER KMW has initialized.
+    $('#message').removeAttr('disabled');
+  },1000);
```

Comment:
Is `setTimeout` really needed here now? Would be *awesome* if we could eliminate it.

Reply:
It _would_ be.  That's the part that needs KMW 13.0.

If KMW's script is the last thing to load on the page, `keyman.init` may not have been called, meaning the touch-aliases won't have been built yet.  If we could wait on `init`'s `Promise`, we could eliminate this timeout in favor of that.

Assuming that making an init call here is safe, granted:  this function doesn't know whatever options the site may be setting elsewhere for its init.

Reply:
So... if we make this 13.0+ then we can eliminate the timeout? Sounds good to me.

Reply:
I've gone ahead and made this into #29.  That way, we get the current batch of work out, and just in case there _is_ any issue with use of init-promises, we can more easily revert just that part.